components/Elements/Select.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { forwardRef } from "react";
+import Select from "react-select";
+import { useTheme } from "@mui/material/styles";
+
+const SelectField = forwardRef((props, ref) => {
```

Comment:
For common components, its a good practice to check types so later new devs or even you can easily use these components.

use [prop-types](https://www.npmjs.com/package/prop-types) library